src/main/java/com/sap/oss/phosphor/fosstars/data/github/VulnerabilitiesFromOwasp.java

```diff
@@ -0,0 +1,279 @@
+package com.sap.oss.phosphor.fosstars.data.github;
+
+import static com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures.VULNERABILITIES;
+import static com.sap.oss.phosphor.fosstars.model.other.Utils.setOf;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sap.oss.phosphor.fosstars.data.AbstractArtifactInfoProvider;
+import com.sap.oss.phosphor.fosstars.data.DataProvider;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.Dependency;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspEntry;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspReference;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.OwaspVulnerability;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.Software;
+import com.sap.oss.phosphor.fosstars.data.github.experimental.owasp.data.VulnerableSoftware;
+import com.sap.oss.phosphor.fosstars.model.Feature;
+import com.sap.oss.phosphor.fosstars.model.ValueSet;
+import com.sap.oss.phosphor.fosstars.model.subject.oss.MavenArtifact;
+import com.sap.oss.phosphor.fosstars.model.value.CVSS;
+import com.sap.oss.phosphor.fosstars.model.value.Reference;
+import com.sap.oss.phosphor.fosstars.model.value.ValueHashSet;
+import com.sap.oss.phosphor.fosstars.model.value.VersionRange;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerabilities;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability.Builder;
+import com.sap.oss.phosphor.fosstars.model.value.Vulnerability.Resolution;
+import java.io.File;
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.owasp.dependencycheck.Engine;
+import org.owasp.dependencycheck.exception.ExceptionCollection;
+import org.owasp.dependencycheck.exception.ReportException;
+import org.owasp.dependencycheck.utils.Settings;
+
+/**
+ * This data provider tries to fill out the
+ * {@link com.sap.oss.phosphor.fosstars.model.feature.oss.OssFeatures#VULNERABILITIES}. This data
+ * provider gathers vulnerabilities about {@link MavenArtifact} using OWASP dependency.
+ */
+public class VulnerabilitiesFromOwasp extends AbstractArtifactInfoProvider<MavenArtifact> {
```

Comment:
OWASP is an organization.

```suggestion
public class VulnerabilitiesFromOwaspDependencyCheck extends AbstractArtifactInfoProvider<MavenArtifact> {
```